sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -496,6 +507,9 @@ def _get_bigquery_client():
             {{ feature }} as {{ featureview.name }}__{{ feature }}{% if loop.last %}{% else %}, {% endif %}
         {% endfor %}
     FROM {{ featureview.table_subquery }}
+    WHERE {{ featureview.event_timestamp_column }} <= '{{ max_timestamp }}' {% if featureview.ttl == 0 %}{% else %}
```

Comment:
For better readability, could move `{% if featureview.ttl == 0 %}{% else %}` to its own line (to be able to match with the `{% endif %}` below).
```
    ...
    WHERE {{ featureview.event_timestamp_column }} <= '{{ max_timestamp }}'
    {% if featureview.ttl == 0 %}{% else %}
    AND {{ featureview.event_timestamp_column }} >= Timestamp_sub(TIMESTAMP '{{ min_timestamp }}', interval {{ featureview.ttl }} second)
    {% endif %}
    ...
```